lib/Parse/ParsePattern.cpp

```diff
@@ -431,6 +431,13 @@ mapParsedParameters(Parser &parser,
                            param.FirstName, param.FirstNameLoc);
     }
 
+    // Warn when an unlabeled parameter follows a variadic parameter
+    if (ellipsisLoc.isValid() && elements.back()->isVariadic() &&
+        param.FirstName.empty()) {
```

Comment:
@CodaFi,

Only the first parameter following a variadic parameter requires a label. I added a test case to verify that:

`func labeledParamFollowingVariadic(_: Any..., label: Any, _: Any) {}
labeledParamFollowingVariadic(1, label: 1, 1)`

Do you still want me to apply it to all unlabeled parameters following a variadic one?